test/extended/templates/templateinstance_security.go

```diff
@@ -109,17 +109,14 @@ var _ = g.Describe("[templates] templateinstance security tests", func() {
 				},
 			},
 			{
-				// at the moment, an admin cannot create a privileged object
-				// via the template instance controller as the latter only has
-				// global edit permissions.
-				by:              "checking adminuser can't create a privileged object",
+				by:              "checking adminuser can create a privileged object",
```

Comment:
hm.  in retrospect, we should probably have a new TC that does fail due to permissions the user has, that the controller doesn't?